docs/genfiles.jl

```diff
@@ -12,7 +12,7 @@ function getfiles()
     srcsfiles = []
     for f in folders
         names = readdir(joinpath(@__DIR__, ".", "examples", f))
-        setdiff!(names, [".DS_Store", "examples_from_esdl_study_4.jl"])
+        setdiff!(names, [".DS_Store", "examples_from_esdl_study_1.jl","examples_from_esdl_study_2.jl","examples_from_esdl_study_3.jl","examples_from_esdl_study_4.jl", "create_from_func.jl"])
```

Comment:
the issue is only `examples_from_esdl_study_1.jl` due to conda/pyplot/pycall

Reply:
also, you will need to remove https://github.com/JuliaDataCubes/YAXArrays.jl/blob/fa679c3715fdae00aa4bd4262dda485446098b7e/.github/workflows/Documenter.yml#L22 until line 24

Reply:
I removed the others because I haven't updated the EarthDataLab package to the DimensionalData syntax yet.

Reply:
mmm... we should not removed them I think unless @meggart says is ok. These a really recent examples. Even the one with pyplot should be replaced and not removed, I will do it later this week. But, the others, your tests should also pass those 😄

Reply:
The problem is not the examples per se, but we would need to bring EarthDataLab to the DimensionalData syntax and this might also be a bit more work. They are only removed until we managed to update EarthDataLab.